docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -176,3 +177,59 @@ For the remaining users:
 
 .. literalinclude:: resource-groups-example.json
     :language: json
+
+.. _scheduleweight-ex:
+
+``schedulingWeight`` example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Schedule weighting is a method of assigning a priority to a resource. Subgroups
+with a higher scheduling weight are given higher priority in order to ensure
+that they are run in a more timely manner.
+
+To support the successful completion of pipelines, you want to weight pipeline
+queries higher than adhoc queries. In the following example, pipelines are
+weighted with a value of ``7``, which is higher than the scheduling weight of
+``3`` that is assigned to adhoc:
```

Comment:
It would be useful to note that with these weights, approximately 70% of your queries will be from the pipeline subgroup, and 30% from the adhoc. Maybe even do the math out, pipeline_percentage = 7/(7 + 3) x 100% = 70%. I've seen a fair number of people get tripped up over the math.